initiatives/2019-09-credsperiment-alpha.json

```diff
@@ -0,0 +1,67 @@
+[
+  {
+    "type": "sourcecred/initiativeFile",
+    "version": "0.2.0"
+  },
+  {
+    "title": "CredSperiment Alpha",
+    "timestampIso": "2019-09-15T00:00:00.000Z",
+    "weight": { "incomplete": 0, "complete": 0 },
+    "completed": false,
+    "champions": ["https://discourse.sourcecred.io/u/decentralion"],
+    "dependencies": {
+      "urls": [
+        "https://discourse.sourcecred.io/t/create-identity-plugin/662",
```

Comment:
The identity plugin was also a dependency. Was mentioned for instance in https://discourse.sourcecred.io/t/introducing-the-credsperiment/183/1's "Pre-requisites"